tests/e2e/matmul/CMakeLists.txt

```diff
@@ -2098,3 +2041,68 @@ iree_generated_e2e_runner_test(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+
+if(IREE_HIP_TEST_TARGET_CHIP MATCHES "^gfx")
+
+unset(IREE_HIP_TEST_COMPILER_FLAGS)
+list(APPEND IREE_HIP_TEST_COMPILER_FLAGS
+  "--iree-rocm-target-chip=${IREE_HIP_TEST_TARGET_CHIP}"
+)
+
+iree_generated_e2e_runner_test(
+  NAME
+    e2e_matmul_rocm_f16_large_cdna3_mfma
```

Comment:
Looks like these are failing now
```
117/124 Test  #163: iree/tests/e2e/matmul/e2e_matmul_rocm_f16_large_cdna3_mfma_rocm_hip ........***Failed    5.54 sec
iree/runtime/src/iree/hal/drivers/hip/native_executable.c:184: FAILED_PRECONDITION; HIP driver error 'hipErrorSharedObjectInitFailed' (303): shared object initialization failed; mismatched target chip? missing/wrong bitcode directory?; while invoking native function hal.executable.create; while calling import; 

The following tests FAILED:
	163 - iree/tests/e2e/matmul/e2e_matmul_rocm_f16_large_cdna3_mfma_rocm_hip (Failed)
	164 - iree/tests/e2e/matmul/e2e_matmul_rocm_f16_large_cdna3_mfma_tb_rocm_hip (Failed)
```

Reply:
Thanks, I updated the compiler flag. Looks like they are passing now.